cardigan/config/colors.js

```diff
@@ -1,27 +1,28 @@
 const colors = {
-  'white': '#ffffff',
-  'black': '#010101',
-  'purple': '#944aa0',
-  'teal': '#006272',
-  'cyan': '#298187',
-  'turquoise': '#5cb8bf',
-  'red': '#e01b2f',
-  'orange': '#e87500',
-  'yellow': '#ffce3c',
-  'brown': '#815e48',
-  'cream': '#f0ede3',
-  'green': '#007868',
-  'charcoal': '#323232',
-  'pewter': '#717171',
-  'silver': '#8f8f8f',
-  'marble': '#bcbab5',
-  'pumice': '#d9d6ce',
-  'smoke': '#e8e8e8',
-  'transparent': 'transparent',
+  white: '#ffffff',
+  black: '#010101',
+  purple: '#944aa0',
+  teal: '#006272',
+  cyan: '#298187',
+  turquoise: '#5cb8bf',
+  red: '#e01b2f',
+  orange: '#e87500',
+  yellow: '#ffce3c',
+  brown: '#815e48',
+  cream: '#f0ede3',
+  green: '#007868',
+  charcoal: '#323232',
+  pewter: '#717171',
+  silver: '#8f8f8f',
+  marble: '#bcbab5',
+  pumice: '#d9d6ce',
+  smoke: '#e8e8e8',
+  coal: '#1f1f1f', // Only to be used for item viewer
+  transparent: 'transparent',
   'transparent-black': 'rgba(29, 29, 29, 0.61)',
   // Opacity value explanation; We use transparent to provide a background to white text which overlays a variety of images (therefore unknown colour contrast).  This opacity is the lightest we can go, while still providing sufficient contrast to pass WCAG guidlines, when it is displayed above a white background, i.e. worst case scenario.
-  'inherit': 'inherit',
-  'currentColor': 'currentColor'
+  inherit: 'inherit',
```

Comment:
What is the reason to do this?

Reply:
IIRC we added `inherit` and `currentColor` to this config so that we could use the `color()` sass function consistently everywhere we used colours in stylesheets.

Given our move away from sass, this is probably no longer something we need to concern ourselves with.

I'm betting the change from `'inherit'` to `inherit` (without quotes) was an autofix by Prettier.